absearch/settings.py

```diff
@@ -146,7 +146,8 @@ def get(self, prod, ver, channel, locale, territory, dist, distver,
             ver = int(ver.split('.')[0])
         except ValueError:
             raise ValueError("Bad version")
-        channel = _lower(channel)
```

Comment:
As in add a test for this new case? Or do we have tests that currently use a channel with a -?